metadata.py

```diff
@@ -1,5 +1,28 @@
 from uuid import uuid5, NAMESPACE_URL
 
+defaults = {
+    'openssh': {
+        'port': 22,
+        'additional_interfaces': [],
+        'only_allow_secure_ciphers': True,
+        'permit_root_login': False,
+        'password_auth': False,
+        'use_pam': False,
+        'gateway_ports': False,
+        'x11': False,
+        'print_motd': True,
+        'use_dns': False,
+        'sign_host_keys': {
+            'enabled': False,
+            'keys': [
+                '/etc/ssh/ssh_host_ed25519_key',
```

Comment:
inconsistent with readme. i like the readme approach better. The full path makes sense for the item, but not for the metadata. would go with the formats, as stated in the readme